coverage.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" ?>
-<coverage version="7.6.8" timestamp="1738187842769" lines-valid="263" lines-covered="147" line-rate="0.5589" branches-covered="0" branches-valid="0" branch-rate="0" complexity="0">
```

Comment:
@davidesner @jordanrburger I think we can exclude this file to be version controlled, what do you think?